app/ui-react/packages/auto-form/src/widgets/TextButton.tsx

```diff
@@ -15,6 +17,7 @@ export class TextButton extends React.Component<ITextButtonProps> {
     const enable =
       typeof this.props.enable !== 'undefined' ? this.props.enable : true;
     const className = this.props.className || '';
+    const id = toValidHtmlId(this.props.id!);
     return (
```

Comment:
With the `extend` suggested above, here all props that are custom to the `TextButton` should be extracted from the props. Something along the lines of:
`const { enable, linkText, className, ...props } = this.props;`